app/Console/Commands/PruneMetaImages.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Console\Commands;
+
+use Illuminate\Console\Command;
+use Illuminate\Support\Facades\DB;
+
+class PruneMetaImages extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'prune-meta-images';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Removes unusued meta images';
+
+    /**
+     * Execute the console command.
+     */
+    public function handle(): int
+    {
+        $validImages = array_map(fn ($item) => $item->image_name, DB::select(get_query('gallery.get_meta_images')));
+
+        $directory = storage_path(sprintf('meta/galleries/'));
+
+        if (is_dir($directory)) {
+            // Open directory
+            if ($handle = opendir($directory)) {
+                // Loop through each file in the directory
+                while (false !== ($file = readdir($handle))) {
+                    // Check if file is a PNG and not in validImages
+                    if (pathinfo($file, PATHINFO_EXTENSION) === 'png' && ! in_array($file, $validImages)) {
```

Comment:
Just to future proof it might be better to use `isset` O(1) because `in_array` potentially has to walk the entire array for each file in the directory O(m*n).

https://stackoverflow.com/a/13483548